File: junior/contest1/skeleton/ninety_nine_skeleton.cpp
#include <algorithm>
#include <iostream>
#include <queue>
#include <sstream>
using namespace std;

int main() {
	for(int n = 0; n < 5; ++n) {
		string line;
		getline(cin, line);
		replace(line.begin(),line.end(),',',' ');
		istringstream sin(line);
		int pts = 0;
		int cards[10] = {};
		sin >> pts;
		for(int i = 0; i < 10; ++i)
			sin >> cards[i];
		//
		// TO_DO: Complete your solution here...
		//
	}
	return 0;
}

File: junior/contest1/skeleton/ninety_nine_skeleton.java
import java.io.*;
import java.util.*;

// TO_DO: either rename the file or the class to make it work...
public class ninety_nine {
	public static void main(String[] args) throws IOException {
		Scanner input = new Scanner(System.in);
		for(int n = 0; n < 5; ++n) { // read 5 input cases
			String line = input.nextLine();
			line = line.replace(',',' ');
			StringTokenizer stk = new StringTokenizer(line);
			int points = Integer.parseInt(stk.nextToken());
			int[] cards = new int[10];
			for(int i = 0; i < 10; ++i)
				cards[i] = Integer.parseInt(stk.nextToken());
			//
			// TO_DO: complete your solution here...
			//
		}
	}
}
